MapboxSpeech/Info.plist

```diff
@@ -15,7 +15,7 @@
 	<key>CFBundlePackageType</key>
 	<string>FMWK</string>
 	<key>CFBundleShortVersionString</key>
-	<string>1.0</string>
+	<string>0.1</string>
```

Comment:
@1ec5 are there other versions I need to change?

Reply:
I’d recommend running `agvtool bump -all` and `agvtool new-marketing-version` as part of any release process.